kubernetes/migration/migrate_test.go

```diff
@@ -145,7 +199,7 @@ func TestMigrate(t *testing.T) {
 			}
 
 			if result != testCase.expectedCorefile {
-				t.Errorf("expected %v; got %v", testCase.expectedCorefile, result)
+				t.Errorf("expected -> got diffs:\n%v", diff.LineDiff(testCase.expectedCorefile, result))
```

Comment:
Is this used due to the order of the `stubDomain` in the expectedCorefile output changing?

Reply:
No, the expected value was getting long, and I couldn't easily spot the difference. So I just use this tool to show which lines were different... turns out it was a white space difference (tab vs space), but knowing which lines it was helped.